acceptance/trivial/chefignore

```diff
@@ -0,0 +1 @@
+.kitchen
```

Comment:
Since we won't be uploading the acceptance suites to a chef server we can safely remove this file